server/datastore/mysql/software_installers.go

```diff
@@ -982,6 +1027,18 @@ WHERE global_or_team_id = ?
 				return ctxerr.Wrapf(ctx, err, "insert new/edited installer with name %q", installer.Filename)
 			}
 
+			// perform side effects if this was an update
+			if len(existing) >= 1 {
```

Comment:
Log warning if `len(existing) > 1`?

Reply:
There's a unique constraint on `global_or_team_id, title_id` so this will always return 0 or 1 rows. At the point where we support multiple installers per team X title we'll need to audit everything, and this will get touched then. Given that info, still worth logging here given that the log will never get hit? Or should I revise to `> 0` to set things at ease?